docs/about/release-notes.rst

```diff
@@ -17,12 +17,13 @@ Features
 * Add ``transform_coords`` for (multi-step) transformations based on existing coords, with support of event coords `#2058 <https://github.com/scipp/scipp/pull/2058>`_.
 * Add ``from_pandas`` and ``from_xarray`` for conversion of pandas dataframes, xarray data arrays and dataset to scipp objects `#2054 <https://github.com/scipp/scipp/pull/2054>`_.
 * Added ``full`` and ``full_like`` variable creation functions `#2069 <https://github.com/scipp/scipp/pull/2069>`_.
-* ``islinspace`` can now take multi-dimensional variables aslong as you pass the dimension to be checked `#2094 https://github.com/scipp/scipp/pull/2094`
+* ``islinspace`` can now take multi-dimensional variables aslong as you pass the dimension to be checked `#2094 https://github.com/scipp/scipp/pull/2094`_.
 * Added a power function and support for the ``**`` operator `#2083 <https://github.com/scipp/scipp/pull/2083>`_.
 * Binned data now has a ``mean`` method as well as ``sum``, which returns the mean of each element within a bin.
 * Add ``scipp.constants`` module for physical constants `#2101 <https://github.com/scipp/scipp/pull/2101>`_.
 * Add ``scipp.spatial.transform`` module providing ``from_rotvec`` and ``as_rotvec`` `#2102 <https://github.com/scipp/scipp/pull/2102>`_.
 * Add ``cross`` function to complementing existing ``dot`` function `#2109 <https://github.com/scipp/scipp/pull/2109>`_.
+* Unary operations such as ``sin`` are now available for datasets as well `#2112 https://github.com/scipp/scipp/pull/2112`_.
```

Comment:
Forgot one thing: Please add `where`

Reply:
Ok, will do in my other PR, so we don't have to rerun builds here.

Reply:
I guess that is fine since `where` isn't really part of either PR